src/app/views/staffs/create-staff-modal/create-staff.html

```diff
@@ -16,24 +16,33 @@ <h3 class="modal-title" translate>mnoe_admin_panel.dashboard.staffs.add_staff.mo
 
       <div class="row top-buffer-1">
         <div class="col-sm-6">
-          <input type="email" class="form-control input-sm" id="user-email" ng-model="vm.user.email" placeholder="{{'mnoe_admin_panel.dashboard.staffs.add_staff.modal.placeholder.email' | translate}}" aria-label="{{'mnoe_admin_panel.dashboard.staffs.add_staff.modal.placeholder.email' | translate}}" required>
+          <input type="email" class="form-control input-sm" id="user-email" ng-model="vm.user.email" ng-change="vm.staffAlreadyExists = false" placeholder="{{'mnoe_admin_panel.dashboard.staffs.add_staff.modal.placeholder.email' | translate}}" aria-label="{{'mnoe_admin_panel.dashboard.staffs.add_staff.modal.placeholder.email' | translate}}" required>
```

Comment:
Best to avoid making an assignment in the template (`ng-change="vm.staffAlreadyExists = false"`), prefer adding an `vm.onEmailChange` method